packages/0xsequence/tests/browser/wallet-provider/dapp.test.ts

```diff
@@ -301,11 +298,11 @@ export const tests = async () => {
 
       // Record wallet deployed state before, so we can check the receipt.to below. We have to do this
       // because a wallet will automatically get bundled for deployment when it sends a transaction.
-      const beforeWalletDeployed = await wallet.isDeployed()
+      const beforeWalletDeployed = await hardhatProvider.getCode(wallet.getAddress()) !== '0x'
```

Comment:
why not keep wallet.isDeployed() method..? its more convienent and better dev experience then `x.getCode(wallet.getAddress()) !== '0x'`

Reply:
The general rationale is that a dapp should never need to "care" about the wallet being deployed or not, this should always be a concern of the wallet only. With EIP-6492 this is even more clear now.

Same with the wallet context / config.

Reply:
I feel like having these methods add a sort of noise when a developer is working with our wallet provider. They may see them and wonder if they need to account for these scenarios or not, and the correct answer is that they don't.